moviesnetwork/moviesdata/services.py

```diff
@@ -0,0 +1,13 @@
+import imdb
+
+from data import Actor
+
+
+def search_actor(name):
+    imdb_access = imdb.IMDb()
+    result = imdb_access.search_person(name)
+    actors = []
+    if result is not None:
+        for r in result:
```

Comment:
I agree that a list comprehension here is better. Also, can `result` be `None`? Is this check necessary? In any case, I prefer doing `if result:`